examples/03-widgets/checkbox-widget.py

```diff
@@ -1,4 +1,5 @@
 """
+.. checkbox_widget_example:
```

Comment:
And we need the preceding underscore here, right? If memory serves that's necessary at the target of the link, but not at the source.

Reply:
That's correct.

```suggestion
.. _checkbox_widget_example:
```